### PR TITLE
Feature/continue mobile cx 2214

### PR DESCRIPTION
**Pull request by elhesuu**
# Problem
Users should see some sort of a "continue on phone" link inside the error message there is a camera exception or  if there is a long period of delay before recording (liveness) / capturing (selfie):
- selfie wait time should be 10s
- video wait time should be 12s

Also, the message should disappear as soon as the user clicks "start recording" in the liveness video case.

# Solution
Since we already have multiple timers in `Photo` and `LivenessCamera`, I tried reusing that logic by encapsulating the timeout creation and clearing when mounting / unmounting. I tried a number of approaches and this one seemed to have a good balance between being reusable and not requiring too many changes.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [] Have new automated tests been implemented?
- [] Have new manual tests been written down?
- [] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/LivenessCamera.js [hunk 1]
```diff
@@ -19,17 +20,20 @@ type Props = {
...
 const initialState = {
   currentIndex: 0,
   isRecording: false,
-  hasTimedOut: false,
+  hasBecomeInactive: false,
+  hasRecordingTakenTooLong: false,
 }
 
-const livenessTimeout = { name: 'LIVENESS_TIMEOUT', type: 'warning' }
+const inactiveError = { name: 'CAMERA_INACTIVE', type: 'warning' }
```
Comment (rfreitas) [inline src/components/Camera/LivenessCamera.js]: Minor: do we have any informal convention for `const` values? like uppercase or something?

Comment (elhesuu) [reply src/components/Camera/LivenessCamera.js]: Only for Redux action types, but not for any others

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Currently blocked by the sub PR: https://github.com/onfido/onfido-sdk-ui/pull/428/files

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
@elhesuu i spotted a issue just now
- on chrome
- go to https://418-pr-onfido-sdk-ui-onfido.surge.sh/?liveness=true
- proceed to liveness
- start recording
- wait 20 seconds
- click redo action
- i get taken to the sms intro screen!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-418-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://418-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**